netlify.toml

```diff
@@ -13,3 +13,11 @@
   to = "/404.html"
   status = 404
   force = false
+
+[[headers]]
+  for = "/*"
+  [headers.values]
+    Strict-Transport-Security = '''
+    max-age=63072000;
```

Comment:
Q: why two years? No objection to doing this (or any value >= 1year), just curious if there's some guidance -- HSTS preload list requires only one year.

Reply:
It's what the Netlify documentation suggested, and it sounded good to me; no other reasoning beyond that.